lectures/pandas.md

```diff
@@ -152,23 +155,9 @@ While a `Series` is a single column of data, a `DataFrame` is several columns, o
 
 In essence, a `DataFrame` in pandas is analogous to a (highly optimized) Excel spreadsheet.
 
-Thus, it is a powerful tool for representing and analyzing data that are naturally organized  into rows and columns, often with  descriptive indexes for individual rows and individual columns.
-
-Let's look at an example that reads data from the CSV file `pandas/data/test_pwt.csv`, which is taken from the Penn World Tables.
+Thus, it is a powerful tool for representing and analyzing data that are naturally organized into rows and columns, often with descriptive indexes for individual rows and individual columns.
 
-Here's the content of `test_pwt.csv`
-
-```{code-block} none
-"country","country isocode","year","POP","XRAT","tcgdp","cc","cg"
-"Argentina","ARG","2000","37335.653","0.9995","295072.21869","75.716805379","5.5788042896"
-"Australia","AUS","2000","19053.186","1.72483","541804.6521","67.759025993","6.7200975332"
-"India","IND","2000","1006300.297","44.9416","1728144.3748","64.575551328","14.072205773"
-"Israel","ISR","2000","6114.57","4.07733","129253.89423","64.436450847","10.266688415"
-"Malawi","MWI","2000","11801.505","59.543808333","5026.2217836","74.707624181","11.658954494"
-"South Africa","ZAF","2000","45064.098","6.93983","227242.36949","72.718710427","5.7265463933"
-"United States","USA","2000","282171.957","1","9898700","72.347054303","6.0324539789"
-"Uruguay","URY","2000","3219.793","12.099591667","25255.961693","78.978740282","5.108067988"
-```
+Let's look at an example that reads data from the CSV file `pandas/data/test_pwt.csv`, which is taken from the [Penn World Tables](https://www.rug.nl/ggdc/productivity/pwt/pwt-releases/pwt-7.0).
 
```

Comment:
@HumphreyYang thanks for linking this to the PWT v7.0. It might be worth adding a small table with the variable names and the description to make it crystal clear. What do you think?